src/coreComponents/unitTests/meshTests/testVTKImport.cpp

```diff
@@ -461,25 +462,36 @@ TEST( VTKImport, medley42 )
 
   auto validate = []( CellBlockManagerABC const & cellBlockManager ) -> void
   {
-    // `medley-42.vtk` is the same as `medley.vtk` with all eleven elements defined as polyhedron.
+    // `supportedElements42.vtk` is the same as `supportedElements.vtk` with all eleven elements defined as polyhedron.
```

Comment:
The name `supportedElements42.vtk` makes perfect sense to me. However, we could also be more verbose and use `supportedElementsAsVTKPolyhedra.vtk`.

Reply:
Yes, it's a little better for people who don't know about VTK file format.

Reply:
Now using `supportedElementsAsVTKPolyhedra.vtk`